contrib/lang/lua/README.md

```diff
@@ -0,0 +1,40 @@
+# Lua contribution layer for Spacemacs
+
```

Comment:
Add a logo, like this one http://www.luteus.biz/Download/LoriotPro_Doc/LUA/LUA_Training_FR/lua.gif
Should be 200 pixels height.

See other READMEs.